src/client/apps/edit/components/admin/components/featuring/featuring_mentioned.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import AutocompleteListMetaphysics from "../../../../../../components/autocomplete2/list_metaphysics"
+import MentionedList from "./mentioned_list"
+
+export const FeaturingMentioned: React.SFC<{ model: string }> = props => {
+  const { model } = props
+
+  const field =
+    model === "artist" ? "primary_featured_artist_ids" : "featured_artwork_ids"
+
+  return (
+    <div>
+      <AutocompleteListMetaphysics field={field} model={`${model}s`} />
```

Comment:
Possibly this `s` is redundant? Could be `featured_artwork_idss`.

Reply:
I can type this so is clearer, model is "artist" or "artwork"

Reply:
Sorry for the confusion - please ignore this. I was looking at the wrong field when I saw the trailing `s`.